src/Language/PureScript/Docs/AsMarkdown.hs

```diff
@@ -0,0 +1,93 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module Language.PureScript.Docs.AsMarkdown where
```

Comment:
No worries. I'm just trying to check off a few things like this with new modules.